plugin.go

```diff
@@ -140,23 +163,40 @@ func ParseOptions(req *plugin_go.CodeGeneratorRequest) (*PluginOptions, error) {
 	}
 
 	parts := strings.Split(params, ",")
-	if len(parts) < 2 || len(parts) > 3 {
+	if len(parts) < 2 {
```

Comment:
Maybe just update the upper bound validation instead of removing it. After this change, we don't want more than 4 options.

Reply:
Good call. Done!